portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+
+import java.io.Console;
+import java.io.IOException;
+import java.util.ArrayList;
+//import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/information")
+public final class InformationServlet extends HttpServlet
+{
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //Strings
+        String currentInfo = "I'm taking finals this week";
+        String pastInfo = "I did a few art commisions this past month" ;
+        String futureInfo = "I'll be aplying to community service programs this summer";
+        //Create array of strings
+        ArrayList<String> personalInfo = new ArrayList<String>();
+        personalInfo.add(currentInfo);
+        personalInfo.add(pastInfo);
+        personalInfo.add(futureInfo);
+        String json = convertToJson(personalInfo);
+        //Display
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+    private String convertToJson(ArrayList<String> info) {
```

Comment:
This is the "stupid" way to do it as the example wants to show you in comparison to gson.convertToJson. You should use gson one instead.

Reply:
Yeah, I did try usign gson but I couldn't get it to work, it was sending the request correctly but it didn't appear on my page for some reason. I'll try to get it to cooperate again and see if it was something else I had changed later.

Reply:
I think you are using it correctly right now, if it works you can remove the commented code. Let me know if you have any question regarding gson.